src/store.js

```diff
@@ -0,0 +1,19 @@
+export const game = {
```

Comment:
why do you still have game object in game.js file... is this one replace it? if it is so you should remove the other one...

Reply:
dont need it anymore. I thought to delete it after merging ...